client/tasks/index.ts

```diff
@@ -2,6 +2,7 @@
  * Internal dependencies
  */
 import { Tasks } from './tasks';
+import './tasks-fills';
```

Comment:
Nitpicky, but can we name this `fills` to keep with the simplified subdirectory naming scheme?